README.md

```diff
@@ -101,6 +101,14 @@ Note: if you do not include the `--config` option with a path to a configuration
 
 To shutdown `mongod` started manually, use the `admin` database and run `db.shutdownServer()`:
 
+ * Shutdown for MongoDB Community Server 6.x and up:
+
+```bash
+mongosh admin --eval "db.shutdownServer()"
+```
+
+ * Shutdown for MongoDB Community Server 5.5 and below:
```

Comment:
sorry to be pedantic, but 5.5 is not an available version for homebrew. This should say 5.0